src/style.css

```diff
@@ -1 +1,14 @@
-/* styles go here */
+.info {
```

Comment:
You should have only classes: for Vertical distance between inputs and Vertical distance between groups